src/main/java/io/ebean/dbmigration/ddlgeneration/BaseDdlHandler.java

```diff
@@ -36,6 +36,11 @@ public void generate(DdlWrite writer, ChangeSet changeSet) throws IOException {
     for (Object change : changeSetChildren) {
       if (change instanceof CreateTable) {
         generate(writer, (CreateTable) change);
+      }
+    }
+    for (Object change : changeSetChildren) {
+      if (change instanceof CreateTable) {
```

Comment:
All create tables must be executed first, otherwise migrations, that depends on an other table will not be executed correctly. (the same may be true also for add column)